cmd/storage-rest-server.go

```diff
@@ -881,15 +873,6 @@ func waitForHTTPStream(respBody io.ReadCloser, w io.Writer) error {
 			}
 			respBody.Close()
 			return errors.New(string(errorText))
-		case 3:
-			// Typed error
-			defer respBody.Close()
-			dec := gob.NewDecoder(respBody)
-			var err error
-			if de := dec.Decode(&err); de == nil {
-				return err
-			}
-			return errors.New("rpc error")
```

Comment:
Preserve this on the client side to avoid storage internode version bump - add a comment to indicate that we should remove this in future @vadmeste

Reply:
> Preserve this on the client side to avoid storage internode version bump - add a comment to indicate that we should remove this in future @vadmeste

yes, good idea, done !